consensus/replay.go

```diff
@@ -474,6 +474,8 @@ func (h *Handshaker) replayBlock(state sm.State, height int64, proxyApp proxy.Ap
 	block := h.store.LoadBlock(height)
 	meta := h.store.LoadBlockMeta(height)
 
+	// Use mocks for both mempool and evidence pool since we don't need them
```

Comment:
Would it be helpful to add an explanation of why we don't need them?

Reply:
sure!